test/e2e/broker_trigger_test.go

```diff
@@ -155,8 +144,9 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 
 	t.Logf("Waiting for triggers to become ready")
 
+	// TODO(chizhg): Change to only waiting for the specified triggers to be ready rather than all.
```

Comment:
I think this will become very important as a single bad test will cause all of them to fail, making it harder to diagnose which was the root cause of the failure.

Not required for this PR, but should be something we fix soon afterwards.

Reply:
No longer a problem after we make each test case running in separate namespace.